### PR TITLE
https where possible

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml [hunk 1]
```diff
@@ -35,12 +35,12 @@
...
-    - deb http://deb.debian.org/debian-security stable/updates main
-    - deb-src http://deb.debian.org/debian-security stable/updates main
+    - deb https://deb.debian.org/debian/ stable main contrib non-free
+    - deb-src https://deb.debian.org/debian/ stable main contrib non-free
+    - deb https://deb.debian.org/debian/ stable-updates main contrib non-free
+    - deb-src https://deb.debian.org/debian/ stable-updates main contrib non-free
+    - deb https://deb.debian.org/debian-security stable/updates main
+    - deb-src https://deb.debian.org/debian-security stable/updates main
     - deb http://ftp.debian.org/debian stretch-backports main
     - deb-src http://ftp.debian.org/debian stretch-backports main
```
Comment (smlambert) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml]: presuming doesn't apply for lines 44-45

Comment (karianna) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml]: correct, this was  manually tested

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 # Additional steps for RedHat and SLES
 - name: Install cpanminus RedHat 6, CentOS 6 and SLES
-  shell: "curl -L http://cpanmin.us | perl - App::cpanminus"
+  shell: "curl -L https://cpanmin.us | perl - App::cpanminus"
```
Comment (sxa555) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml]: Hopefully curl on all our boxes is SSL-enabled :-)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml [hunk 3]
```diff
@@ -26,14 +26,14 @@
 - name: Import AZUL public key
   rpm_key:
     state: present
-    key: http://repos.azulsystems.com/RPM-GPG-KEY-azulsystems
+    key: https://repos.azulsystems.com/RPM-GPG-KEY-azulsystems
```
Comment (sxa555) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml]: Seems to have a certificate that may not work in automation - hostname mismatch

```
repos.azulsystems.com uses an invalid security certificate. The certificate is only valid for the following names: *.cloudfront.net, cloudfront.net 
```

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml (hunk 2 above)
Comment (karianna) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml]: If they don't then we've got bigger problems :-)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml (hunk 3 above)
Comment (karianna) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml]: Yep my bad, back to http you go, will push in a sec

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml [hunk 4]
```diff
@@ -35,12 +35,12 @@
...
-    - deb-src http://deb.debian.org/debian/ stable-updates main contrib non-free
-    - deb http://deb.debian.org/debian-security stable/updates main
-    - deb-src http://deb.debian.org/debian-security stable/updates main
+    - deb https://deb.debian.org/debian/ stable main contrib non-free
+    - deb-src https://deb.debian.org/debian/ stable main contrib non-free
+    - deb https://deb.debian.org/debian/ stable-updates main contrib non-free
+    - deb-src https://deb.debian.org/debian/ stable-updates main contrib non-free
+    - deb https://deb.debian.org/debian-security stable/updates main
+    - deb-src https://deb.debian.org/debian-security stable/updates main
     - deb http://ftp.debian.org/debian stretch-backports main
```
Comment (sxa555) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml]: I know it's not part of this PR, but that URL gives a 404 at the moment ...

**Pull request merged**